Loop/Info.plist

```diff
@@ -2,6 +2,15 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>Loop</string>
```

Comment:
Lowercase would be preferred. Any idea if this will create conflicts with another app? 

Alternatively you could derive something unique using the bundle identifier, which will be nice for the few folks who have multiple installations.